extras/sponge.rb

```diff
@@ -141,7 +141,7 @@ def fetch(url, method = :get, fields = nil, raw_post_data = nil, headers = {}, l
     end
 
     if BAD_NETS.select {|n| IPAddr.new(n).include?(ip) }.any?
-      raise BadIPsError.new("refusing to talk to IP #{ip}")
+      raise BadIPsError.new("refusing to talk to IP #{ip}") if Rails.env.production?
```

Comment:
What triggered this in dev/testing? I'm leery of relaxing this protection in such a general way.

Reply:
this blocks requests to localhost, so i couldn't test an end-to-end proof locally (with a locally running keybase stack) without suppressing the check in dev. i understand your concern. i'll remove it and replace with a little comment somewhere else.